be/src/olap/comparison_predicate.cpp

```diff
@@ -548,8 +547,7 @@ COMPARISON_PRED_SET_DICT_CODE(NotEqualPredicate)
                 auto& dict_col =                                                               \
                         reinterpret_cast<vectorized::ColumnDictionary<vectorized::Int32>&>(    \
                                 *col_ptr);                                                     \
-                auto code = dict_col.find_code_by_bound(_value, 0 OP 1, 1 OP 1);               \
-                _dict_code = code;                                                             \
+                _dict_code = dict_col.find_code_by_bound(_value, 1 OP 0, 1 OP 1);              \
```

Comment:
what means of 1,0 ? maybe we need more comment?

Reply:
If `1 OP 0` returns true, it means the predicate is `>` or `>=`,
If `1 OP 1` returns true, it means the predicate is `>=` or `<=`
Through this trick, avoid redundant code, such as `bool is_greater = type() == PredicateType::GT || type() == PredicateType::GE;`
ok, i will add a comment.